manager/src/main/java/com/redhat/service/smartevents/manager/api/models/requests/BridgeRequest.java

```diff
@@ -1,28 +1,45 @@
 package com.redhat.service.smartevents.manager.api.models.requests;
 
+import javax.validation.Valid;
 import javax.validation.constraints.NotEmpty;
 
 import com.fasterxml.jackson.annotation.JsonProperty;
+import com.redhat.service.smartevents.infra.models.gateways.Action;
+import com.redhat.service.smartevents.manager.api.user.validators.processors.ValidErrorHandler;
 import com.redhat.service.smartevents.manager.models.Bridge;
 
+@ValidErrorHandler
 public class BridgeRequest {
 
     @NotEmpty(message = "Bridge name cannot be null or empty")
     @JsonProperty("name")
     private String name;
 
+    @JsonProperty("errorHandler")
+    @Valid
+    private Action errorHandler;
```

Comment:
Can you please update readme/doc to cover error handling too? Like [`DEMO.md`](https://github.com/5733d9e2be6485d52ffa08870cabdee0/sandbox/blob/main/DEMO.md)

Reply:
done :+1: